src/dune_rules/pkg_rules.ml

```diff
@@ -467,10 +472,16 @@ module Run_with_path = struct
         | Path p -> Path (f p))
     ;;
 
-    let bimap t f _g = { t with args = Array.Immutable.map t.args ~f:(map_arg ~f) }
+    let bimap t f _g =
+      { t with
+        args = Array.Immutable.map t.args ~f:(map_arg ~f)
+      ; set_ocamlfind_destdir = Option.map ~f t.set_ocamlfind_destdir
+      }
+    ;;
+
     let is_useful_to ~memoize:_ = true
 
-    let encode { prog; args } path _ : Dune_lang.t =
+    let encode { prog; args; set_ocamlfind_destdir = _ } path _ : Dune_lang.t =
```

Comment:
might as well add it to the digest just in case

Reply:
done